core/src/HIP/Kokkos_HIP_Graph_Impl.hpp

```diff
@@ -69,6 +69,9 @@ class GraphImpl<Kokkos::HIP> {
   template <class... PredecessorRefs>
   auto create_aggregate_ptr(PredecessorRefs&&...);
 
+  const hipGraph_t     getNativeGraph()           const { return m_graph; }
+  const hipGraphExec_t getNativeExecutableGraph() const { return m_graph_exec; }
```

Comment:
Matching the local style
```suggestion
  const hipGraph_t     get_native_graph()           const { return m_graph; }
  const hipGraphExec_t get_native_executable_graph() const { return m_graph_exec; }
```

Reply:
Indeed ! Done :heavy_check_mark: